supplements/candlekeep-mysteries/items.xml

```diff
@@ -0,0 +1,270 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<elements>
+	<info>
+		<name>Items</name>
+		<description>The Items from Candlekeep Mysteries</description>
+		<author url="http://dnd.wizards.com/products/tabletop-games/rpg-products/candlekeep-mysteries">Candlekeep Mysteries</author>
+		<update version="0.0.1">
+			<file name="items.xml" url="https://raw.githubusercontent.com/AuroraLegacy/elements/master/supplements/candlekeep-mysteries/items.xml" />
+		</update>
+	</info>
+	<element name="Alchemy Jug (Blue)" type="Magic Item" source="Candlekeep Mysteries" id="ID_WOTC_CM_MAGIC_ITEM_ALCHEMY_JUG_BLUE"> <!--pg 144-->
```

Comment:
are we starting to comment which page item can be found on?

Reply:
I had the info so figured it wouldn't hurt since the items are all over the book.  I don't think its worth going back and adding them elsewhere.